pyproject.toml

```diff
@@ -15,6 +15,9 @@ classifiers = [
 python = ">=3.9.0,<3.11"
 fastapi = "0.100.1"
 uvicorn = {version = "0.23.2", extras = ["standard"]}
+fl4health = "^0.1.11"
+wandb = "^0.16.3"
+torchvision = "0.14.1"
```

Comment:
Are `wandb` and `torchvision` FL4Health's dependencies? I yes, we should consider changing the dependencies list there. Not for this PR though.

Reply:
Yeah that is strange! I just checked and wandb and torchvision are dev dependencies. But I am pretty sure they should be normal dependencies, I will make a not to update on the fl4health side and put a PR

Reply:
Thanks!